compoments/radioFieldSet.js

```diff
@@ -53,17 +54,36 @@ class RadioFieldSet extends Component {
             {this.state.error}
           </span>
           <div className="govuk-radios" onChange={this.setValue.bind(this)}>
-            {this.options.map((o, i) => (
-              <div className="govuk-radios__item" key={i}>
-                <input className="govuk-radios__input govuk-input--width-10"
-                  id={`${this.name}-${i}`} name={this.name}
-                  type="radio" value={o.value} defaultChecked={o.checked}/>
-                <label className="govuk-label govuk-radios__label"
-                  htmlFor={`${this.name}-${i}`}>
-                  {o.title}
-                </label>
-              </div>
-            ))}
+            {this.options.map((o, i) => {
+              if (this.includeOrDivider && i == (this.options.length - 1)) {
```

Comment:
This check is being performed for every item, but it'll only be true for the last item - consider refactoring to improve flow of code.